runtime/extension/extension_helper.cpp

```diff
@@ -20,8 +20,10 @@
 #include <vector>
 
 #ifdef BPFTIME_ENABLE_IOURING_EXT
+#ifdef USE_BPF
```

Comment:
`__linux__` instead? uring is specific to linux

Reply:
I have added your suggestion